src/index.ts

```diff
@@ -4,6 +4,7 @@ import ResultsCalculator from './services/results-calculator';
 
 /*
 	If input file defined in command line, use as input.
+	Otherwise, we will use process.stdin.
 */
 const inputFile = process.argv[2];
```

Comment:
In case there are more arguments passed into the command line, this should be more declarative.

I would suggest using a library like 'yargs', and then enforcing a variable name (e.g. '--file <path-to-file>') on the command line, and using yargs to parse the 'file' variable's value. 

This will make this more extensible in the future in case users need to pass other variables on the command line as well (and they can do so in any order they wish).